symbols/Fernmeldewesen/Bedingung_DMO.j2

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" xmlns="http://www.w3.org/2000/svg" height="256">
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="256" height="256" viewbox="0 0 256 256">
```

Comment:
Die selbe Anpassung ist schon in #150 enthalten, ggf. hier reverten um merge conflict vorzubeugen?

Reply:
> Die selbe Anpassung ist schon in #150 enthalten, ggf. hier reverten um merge conflict vorzubeugen?

Ich vermute du meinst #152. Wenn #152 vor diesem PR integriert wird, wäre ein Revert unproblematisch, andernfalls wäre nach dem Merge dieses PR taktische Zeichen "Bedingung_DMO" nicht als brauchbare PNG-Version verfügbar.